src/topology/subset_properties.lean

```diff
@@ -1118,6 +1118,25 @@ lemma subtype.connected_space {s : set α} (h : is_connected s) :
   (subtype.preconnected_space h.is_preconnected).is_preconnected_univ,
   to_nonempty := h.nonempty.to_subtype }
 
+lemma is_preconnected_iff_preconnected_space {s : set α} :
+  is_preconnected s ↔ preconnected_space s :=
```

Comment:
I think that `←` is `by simpa using is_preconnected_univ.image (??name?? : continuous coe)`.